src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,86 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int INITIAL_CAPACITY = 1 << 4;
+    static final double LOAD_FACTOR = 0.75;
+    static final int RESIZE_MULTIPLAYER = 2;
+    private int size;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        table = new Node[INITIAL_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
+        if (getSize() >= table.length * LOAD_FACTOR) {
```

Comment:
```suggestion
        if (size >= table.length * LOAD_FACTOR) {
```
we don't need to use getter if we can use field
check all file and fix everywhere